src/Microsoft.AspNet.SignalR.Redis/RedisConnection.cs

```diff
@@ -17,7 +17,11 @@ public class RedisConnection : IRedisConnection
 
         public async Task ConnectAsync(string connectionString, TraceSource trace)
         {
-            _connection = await ConnectionMultiplexer.ConnectAsync(connectionString);
+            _connection = await ConnectionMultiplexer.ConnectAsync(connectionString, new TraceTextWriter("ConnectionMultiplexer: ", trace));
+            if(!_connection.IsConnected)
+            {
```

Comment:
Well, when it does "re-connect", the object actually becomes active again, so maybe not.